Source/types/PrimitiveTypeMap.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+export interface PrimitiveTypeMap {
+    string: string;
+    number: number;
+    boolean: boolean;
```

Comment:
Dont really know what this is, but should bigint be here as well?

Reply:
Yes. Doing it

Reply:
Fixed by https://github.com/dolittle-fundamentals/JavaScript.Fundamentals/pull/9/commits/be4275d0cb8ea8773d143eef3c8e808b20e42f8c